dev/tests/integration/testsuite/Magento/Wishlist/Controller/Index/CartTest.php

```diff
@@ -69,7 +69,8 @@ public function testAddSimpleProductToCart(): void
         $item = $this->getWishlistByCustomerId->getItemBySku(1, 'simple-1');
         $this->assertNotNull($item);
         $this->performAddToCartRequest(['item' => $item->getId(), 'qty' => 3]);
-        $message = sprintf('You added %s to your shopping cart.', $item->getName());
+        $message = sprintf("\n" . 'You added %s to your ' .
```

Comment:
Why do we need new like character here at the beginning ?

Reply:
I think this is due to the using of "addComplexSuccessMessage".
It looks like we can see in 
app/code/Magento/Checkout/Controller/Cart/Add.php
![image](https://user-images.githubusercontent.com/51681435/89281650-10c1fe00-d653-11ea-8799-147e61d1a4b9.png)
dev/tests/integration/testsuite/Magento/Checkout/Controller/CartTest.php
![image](https://user-images.githubusercontent.com/51681435/89281680-1b7c9300-d653-11ea-82cf-be681b1bbc5f.png)

Additional research is needed here.

Reply:
Ah, inside test it's ok. No need to investigate it